servers/src/main/java/tachyon/master/next/filesystem/meta/InodeTree.java

```diff
@@ -220,7 +220,7 @@ public void deleteInode(Inode inode) throws FileDoesNotExistException {
 
     mInodes.remove(inode);
     mPinnedInodeFileIds.remove(inode.getId());
-    inode.reverseId();
+    inode.delete();
```

Comment:
O ok. I meant that the TODO could just be to investigate if anything ever checks "id < 0". You don't have to change anything now.